ibis/expr/api.py

```diff
@@ -3108,3 +3199,26 @@ def _table_drop(self, fields):
 
 
 _add_methods(ir.TableExpr, _table_methods)
+
+
+# geometric operation
+
+def distance(from_lon, from_lat, to_lon, to_lat):
+    """
+    Distance between origin longitude and latitude and
+    destine longitude and latitude
```

Comment:
It looks like `destine` is actually an English word (which I did not know), but I wonder if this reads more clearly as "destination" instead